src/coreclr/vm/methoditer.cpp

```diff
@@ -108,9 +108,6 @@ BOOL LoadedMethodDescIterator::Next(
 
```

Comment:
Remove the comment at line 100 around being really careful in the presence of unrestored types.